docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -86,6 +86,7 @@ give it a try.
 ### Deploying with command-line (recommended method)
 
 You can build and deploy with the following command:
+`.\Restore.cmd -Configuration Release` followed by
 `.\Build.cmd -Configuration Release -deployExtensions -launch`.
```

Comment:
```suggestion
`.\Build.cmd -Restore -Configuration Release -deployExtensions -launch`.
```

Why two lines here? This one line should do the same thing.

Reply:
That's better indeed. Thanks